### DISCUSSION TITLE
how to get the jobId, cluster name and job status easily when i was running the job with seatunnel engine.

### DISCUSSION THREAD

**Discussion by ggww** [apache/seatunnel#4344] [Q&A]
when i was running the kafka to mysql seatunnel stream job through the seatunnel job, i can see the realtime log,and there has the jobId, cluster name, job status.  whether to have the shell to get it or other methods.

if the job is batch, how to get them (jobId, cluster name, job status) .

**Comment by liunaijie** [comment]
now, there has cli and rest-api to get those infos
